src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/display/DisplayHandler.kt

```diff
@@ -20,12 +21,31 @@ object DisplayHandler {
     fun clearDisplays() = displays.clear()
 
     @SubscribeEvent
-    fun renderDisplays(event: RenderGameOverlayEvent.Text) {
+    fun renderDisplayOverlay(event: RenderGameOverlayEvent.Text) {
         GlStateManager.pushMatrix()
-        displays.forEach(Display::render)
+        displays.forEach {
+            if(it.registerType == RegisterType.RENDER_OVERLAY) {
+                it.render()
+            }
+        }
         GlStateManager.popMatrix()
     }
 
+    @SubscribeEvent
+    fun renderDisplayGui(event: GuiScreenEvent.DrawScreenEvent.Post) {
+        GlStateManager.pushMatrix()
+        displays.forEach {
+            if(it.registerType == RegisterType.POST_GUI_RENDER) {
```

Comment:
There should be a space after if

Reply:
Alright, I assume that's all the spelling checks you have correct?